src/pipecat/services/aws/stt.py

```diff
@@ -284,9 +284,7 @@ async def _receive_loop(self):
                 break
 
             try:
-                response = await self._ws_client.recv()
-
-                self.start_watchdog()
+                response = await asyncio.wait_for(self._ws_client.recv(), timeout=1.0)
```

Comment:
Same here, I guess we need to handle the `TimeoutError` in this case now, right ?